### DISCUSSION TITLE
List of all HOMEBREW environment variables?

### DISCUSSION THREAD

**Discussion by halloleo** [Homebrew/discussions#2947] [Everyday usage] [answered]
Is there a list of all HOMEBREW environment variables somewhere in the docu? 

In various places I found things like `HOMEBREW_NO_EMOJI` and `HOMEBREW_VERBOSE`, but a list of `all` variables would prove invaluable I think. (But of cpurse such a list is a nightmare to keep up-to-date...)

**Comment by gromgit** [comment, marked as answer]
It's in the man page: https://docs.brew.sh/Manpage#environment or `man brew`.

**Reply by halloleo** [reply]
Thanks for this - and sorry: I should have seen that myself.
